src/components/HTMLEngineProvider/HTMLRenderers/NextStepsEmailRenderer.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
```

Comment:
Cna you make this file in TS?

Reply:
Yes, I can. It is good news! I just though that `src/component` still should be JS.

Reply:
Done - 858e6ed.